src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,110 @@
+.card {
+  font-family: Roboto, sans-serif;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  box-sizing: border-box;
```

Comment:
You shouldn't use `*` selector for `margin` and `padding`, because it impacts the performance. 

But what about `box-sizing`? 🤔

In case you have 30 different blocks on the page, will you write `box-sizing` to all these blocks? In a big project if somebody forgot to add `box-sizing` it can be hard to debug in the future.

So it's good to write `box-sizing` in `*` selector. Let's move it there.